jax/_src/config.py

```diff
@@ -794,6 +794,13 @@ def update_thread_local_jit_state(**kw):
           'computations. Logging is performed with `logging` at WARNING '
           'level.'))
 
+raw_prng_key = config.define_enum_state(
```

Comment:
```suggestion
legacy_prng_key = config.define_enum_state(
```

I'm not sure everyone will understand what "raw" means. What do you think of this? Other terms: `uint32`, or `numeric`. See other comments, relatedly.

Reply:
Changed to `legacy` – thanks